src/helpers/get-ogc-items-stream.ts

```diff
@@ -0,0 +1,25 @@
+import { PagingStream } from '../paging-stream';
+import { getPagingStream } from './get-paging-stream';
+import * as _ from 'lodash';
+import QueryString from 'qs';
+export const getOgcItemsStream = 
+  async (siteUrl: string, collectionKey: string, requestQuery: QueryString.ParsedQs): Promise<PagingStream> => {
+  const requestParams = buildSearchRequestParams(siteUrl, collectionKey, requestQuery);
+  // OGC API can only return 100 results at maximum so
```

Comment:
Let's discuss this in a conversation. I don't totally understand the ramifications of this choice / why we can only get 100.

Reply:
Per our conversation, the comment imply that if a request query specify a limit, 100 is the max we can do due to current OGC Search API limitation. I have updated the comment.